### PR TITLE
Add `nrfutil`

### PR DESCRIPTION
**Pull request by fgervais**
This is useful to program devices like `nrf52840dongle_nrf52840` using the image.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -56,6 +56,7 @@ RUN mkdir /workdir/project && \
     python3 -m pip install -U setuptools && \
     python3 -m pip install cmake>=3.20.0 wheel && \
     python3 -m pip install -U west==0.12.0 && \
+    python3 -m pip install -U nrfutil==6.1.3 && \
```
Comment (coderbyheart) [inline Dockerfile]:
```suggestion
    python3 -m pip install -U nrfutil && \
```
**question:** Any reason to pin the version, why not use the latest one?

Comment (fgervais) [reply Dockerfile]: I have a habit of pinning versions as much as I can as it gives me better reproducibility between image builds. Also I saw you pinned some other packages so I did too.

In this case, I don't see any strong requirements for pinning. I will change as you suggests.

Thank you for the prompt review!

**Comment by coderbyheart** [comment]
The ones that are pinned are because this specific version is needed. I'd avoid adding more pinned versions because we have to update them by hand.

**Comment by coderbyheart** [comment]
Thank you!

**Pull request merged**